components/ResultsPage/index.tsx

```diff
@@ -18,25 +21,55 @@ export const ResultsPage: React.VFC = () => {
   const root = useStore()
   const router = useRouter()
 
+  const listLinks = [
+    { text: tsln.resultsPage.youMayBeEligible, url: "#eligible" },
+    { text: tsln.resultsPage.yourEstimatedTotal, url: "#estimated" },
+    { text: tsln.resultsPage.whatYouToldUs, url: "#answers" },
+    { text: tsln.resultsPage.nextSteps, url: "#next" },
+    { text: tsln.resultsPage.youMayNotBeEligible, url: "#noteligible" }
+  ]
+
+  console.log("Root inputs proxy")
+  console.log(root.inputs)
+  const inputX = root.inputs
+
+  
+  inputX.map(([key,value]) => {
+    console.log('key =',key,'===',tsln.resultsQuestions[key],'| value=', value)
+  }) 
+
+  const xxx = <span dangerouslySetInnerHTML={{ __html: root.summary.details }} ></span>
```

Comment:
please use a relevant variable name

Reply:
this is temp work around until Anthony creates a new version of the summary component